cmd/server/cadence.go

```diff
@@ -53,7 +54,12 @@ func startHandler(c *cli.Context) {
 	log.Printf("config=\n%v\n", cfg.String())
 
 	cassCfg := cfg.Cassandra
-	if err := cassandra.VerifyCompatibleVersion(cassCfg); err != nil {
+	dir, err := os.Getwd() // dir = /Users/madhuravi/Uber/gocode/src/github.com/uber/cadence
```

Comment:
how about we just change our schema update process to also update a constant in version.go ? that would be simple and less brittle going forward

Reply:
I know I brought up  that discussion again but I'm not so convinced about it. It does not introduce much risk since it is the same path, we will know early in local testing etc. if this breaks. It's more risky if it somehow breaks in production without us knowing but I can't see that happening.

Reply:
ok, will leave it to you - my preference is always for the const in code - its super straightforward - but this is something we can change it when we need it

Reply:
Agreed. Will try this out now, open to changing it based on what works, can see the value of both.